### DISCUSSION TITLE
[Feature] Auto-add X face to Y Album

### DISCUSSION THREAD

**Discussion by tristenlammi** [immich-app/immich#12154] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to select certain faces to be automatically added to an album would be incredible. For example, we have an album called "Bumi" for our dog. It'd be great if Immich could auto add any photos with bumi in it to that album.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by radh21301** [comment]
This is a master feature that includes what you want.

https://github.com/immich-app/immich/discussions/1673

If that does cover your use case, please close this request.
